modules/tinymce/src/plugins/template/test/ts/module/InsertTemplate.ts

```diff
@@ -0,0 +1,22 @@
+import { UiFinder, Waiter } from '@ephox/agar';
+import { TinyUiActions } from '@ephox/mcagar';
+import { SugarBody, SugarElement } from '@ephox/sugar';
+
+import Editor from 'tinymce/core/api/Editor';
+
+const toolbarButtonSelector = '[role="toolbar"] button[aria-label="Insert template"]';
+const dialogSelector = 'div.tox-dialog';
+
+const insertTemplate = async (editor: Editor, assertFn?: (elm: SugarElement<Node>) => Promise<void>) => {
```

Comment:
Per naming conventions, this should be called `pInsertTemplate`.

- [ ] Rename this function